msteams-platform/toolkit/TeamsFx-SDK.md

```diff
@@ -10,25 +10,28 @@ ms.date: 11/29/2021
 # TeamsFx SDK
 
-TeamsFx helps to reduce the developer tasks by using Teams SSO and accessing cloud resources down to single line statements with zero configuration. TeamsFx SDK is built to be used in browser and Node.js environment, common scenarios include:
+TeamsFx helps to reduce your tasks by using Microsoft Teams Single-Sign-On (Teams SSO) and accessing cloud resources down to single line statements with zero configuration. You can use TeamsFx SDK in browser and Node.js environment. TeamsFx core functionalities can be accessed in client as well as server environment and you can write user authentication code in a simplified way for:
 
-* Teams tab application
-* Azure Function
+* Teams tab
 * Teams bot
-
-You can use the TeamsFx SDK to:
-
-* Access the core functionalities in client and server environment.
-* Write user authentication code in a simplified way.
+* Azure function
 
 ## Prerequisites
 
-Install the following tools and set up your development environment:
+You need to install the following tools and set up your development environment:
+
+| &nbsp; | Install | For using... |
+   | --- | --- | --- |
+   | &nbsp; | [Visual Studio Code](https://code.visualstudio.com/download) | JavaScript, TypeScript, or SharePoint Framework (SPFx) build environments. Use version 1.55 or later. |
+   | &nbsp; | [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)| A Microsoft Visual Studio Code extension that creates a project scaffolding for your app. Use 4.0.0 version. |
+   | &nbsp; | [Node.js](https://nodejs.org/en/download/) | Back-end JavaScript runtime environment. Use the latest v16 LTS release.|
+   | &nbsp; | [Microsoft Teams](https://www.microsoft.com/microsoft-teams/download-app) | Microsoft Teams to collaborate with everyone you work with through apps for chat, meetings, call - all in one place.|
+   | &nbsp; | [Microsoft&nbsp;Edge](https://www.microsoft.com/edge) (recommended) or [Google Chrome](https://www.google.com/chrome/) | A browser with developer tools. |
 
-* Latest version of Node.js
-* If your project has installed `botbuilder` related [packages](https://github.com/Microsoft/botbuilder-js#packages) as dependencies, ensure they are of the same version. Currently, the required version is 4.15.0 or later, for more information, see [bot builder packages should be of the same version](https://github.com/BotBuilderCommunity/botbuilder-community-js/issues/57#issuecomment-508538548).
+> [!NOTE]
+> If your project has installed `botbuilder` related [packages](https://github.com/Microsoft/botbuilder-js#packages) as dependencies, ensure they are of the same version. Currently, the required version is 4.15.0 or later, for more information, see [bot builder packages should be of the same version](https://github.com/BotBuilderCommunity/botbuilder-community-js/issues/57#issuecomment-508538548).
 
-You must have working knowledge of the following:
+You must have working knowledge of:
 
```

Comment:
Those are more for "reference" types of things, shall we consider to move it to the bottom?

Reply:
Will confirm with Abira and implement, once agreed.

Reply:
This cannot be moved to the bottom of the page as its related to perquisite installation.

Reply:
But I saw sample link is also listed on the bottom